.eslintrc.cjs

```diff
@@ -0,0 +1,54 @@
+module.exports = {
```

Comment:
Do we now why this had to be changed to a .cjs? I'm just generally curious.

Reply:
It's using `module.exports` so the linter complains when it sees a file doing that:

```
Error [ERR_REQUIRE_ESM]: require() of ES Module /Users/selkie/Development/commonwealth/.eslintrc.js from /Users/selkie/Development/commonwealth/node_modules/@eslint/eslintrc/dist/eslintrc.cjs not supported.
.eslintrc.js is treated as an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which declares all .js files in that package scope as ES modules.
Instead rename .eslintrc.js to end in .cjs, change the requiring code to use dynamic import() which is available in all CommonJS modules, or change "type": "module" to "type": "commonjs" in /Users/selkie/Development/commonwealth/package.json to treat all .js files as CommonJS (using .mjs for all ES modules instead).
```

It's not obvious why Git doesn't pick up that the file was renamed - there aren't any changes to the file contents.